packages/dashboard-frontend/src/components/ImportFromGit/index.tsx

```diff
@@ -180,7 +180,7 @@ class ImportFromGit extends React.PureComponent<Props, State> {
           validated={validated}
           helperTextInvalid={errorMessage}
           helperTextInvalidIcon={<ExclamationCircleIcon />}
-          helperText="Import from a Git repository to create your first workspace."
+          helperText="Import a devfile from a Git repository to create a new workspace or launch an existing one."
```

Comment:
```suggestion
          helperText="Import from a Git repository to create a Cloud Development Environment"
```

Reply:
@tyraziel @olexii4 wdyt?

Reply:
I think my main hangup is that if an environment of the same name already exists, it will ask if you want to open the existing one or create a new one.  Trying to capture that concisely is the tricky part.

What if we changed it from "create" to "launch"?  Does launch indicate create/open?

"Import from a Git repository to launch a Cloud Development Environment"

@ibuziuk @olexii4 ?

Reply:
@tyraziel "launch" sounds good 👍

Reply:
Committed update.